<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="13" column="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>PakType Naqsh</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Arnold Lights</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pushButton_8">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Clear Selection</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Reload Lights</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>PakType Naqsh</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Maya Lights</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
